### ISSUE TITLE
Something like EntityPlayer:GetDamageMultiplier() should be added

### ISSUE THREAD

**Issue by TiredFoks** [Meowlala/RepentanceAPIIssueTracker#491]
A function that just returns the current damage multiplier would be very handy.

**Comment by TiredFoks** [comment]
Also it could be divided into two functions for item multipliers and character specific multipliers.

**Comment by Zamiell** [comment]
there is no such thing as a damage multiplier.
damage is calculated on a continuous basis from start to finish.
for example, picking up sacred heart after already having magic mushroom does not "change your damage multiplier".
it just increases your damage because an additional if statement is evaluated.

**Comment by oatmealine** [comment]
not quite; read https://bindingofisaacrebirth.fandom.com/wiki/Damage#Damage_Multipliers for more info but the system is quite a bit more complicated than that:
> Some items multiply damage rather than simply adding to it. Damage multipliers do not simply multiply the character's damage once; the damage stat is constantly recalculated and the damage multipliers are multiplied with the end result.

**Comment by Zamiell** [comment]
the wiki's formula does not reflect what the actual C++ code is doing
